config/deploy.rb

```diff
@@ -17,31 +16,30 @@
   color: :auto,
   truncate: :auto
 
-# Default value for :pty is false
-# set :pty, true
-
 set :linked_files, fetch(:linked_files, [])
   .push(
     'server/.env'
   )
 
 set :linked_dirs, fetch(:linked_dirs, [])
   .push(
-    'log',
+    'server/storage/logs',
```

Comment:
That's `log/`, and that's a folder that Capistrano generates during deployments that fail.